Tasks/XamarinAndroid/XamarinAndroid.ps1

```diff
@@ -42,6 +42,15 @@ try {
     if ($outputDir) {
         $msBuildArguments = "$msBuildArguments /p:OutputPath=""$outputDir"""
     }
+    
```

Comment:
Why not use the method you added to java-common? you added the reference in make.json.